rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMStateCaching.java

```diff
@@ -40,60 +40,37 @@
 
 
 /**
- * Periodically request {@link RMProxyUserInterface#getMonitoring()} and store it locally.
- * <p>
- * The {@link RMInitialState} fetched from {@link RMProxyUserInterface} is a large object
- * that is long to serialize, but is always the same for every client.
- * <p>
- * Use this class to start a thread that will periodically get this object
- * using a watcher account, making the cached version available to any client instantly.
- * <p>
- * Refresh rate can be configured using {@link PortalConfiguration#RM_CACHE_REFRESHRATE}
+ * Creates new AO {@link RMProxyUserInterface} in order to access filtered events.
  */
 public class RMStateCaching {
 
     private static Logger logger = ProActiveLogger.getLogger(RMStateCaching.class);
 
     private static RMProxyUserInterface rm;
 
-    private static RMInitialState state;
-
-    private static Thread rmUpdater;
-
-    private static int refreshInterval;
-
-    private static boolean kill = false;
+    private static RMProxyUserInterface aorm;
 
     /**
      * Start a thread that will periodically fetch {@link RMProxyUserInterface#getMonitoring()}.
      * <p>
      * Thread frequency can be customized using {@link PortalConfiguration#RM_CACHE_REFRESHRATE}.
      * <p>
-     * Cached object can be retrieved using {@link #getRMInitialState()}.
-     * <p>
-     * Stop this thread by calling {@link #kill()}.
+     * Cached object can be retrieved using {@link #getRMInitialState(long)}
      */
     public synchronized static void init() {
-        new Thread(new Runnable() {
-            @Override
-            public void run() {
-                state = new RMInitialState();
-                init_();
-                run_();
-            }
-        }).start();
+        init_();
     }
 
     private static void init_() {
-        refreshInterval = PortalConfiguration.RM_CACHE_REFRESHRATE.getValueAsInt();
-
-        while (rm == null) {
+        while (aorm == null) {
```

Comment:
can you give a better name instead of aorm ?

Reply:
what can be better then 4 small characters abbreviation. 
but i will try to make it even better